pyof/v0x04/common/flow_match.py

```diff
@@ -213,7 +207,7 @@ def __init__(
         super().__init__()
         self.oxm_class = oxm_class
         self.oxm_field_and_mask = None
-        self.oxm_length = oxm_length
+        self.oxm_length = None
```

Comment:
this is reverting changes from #109 (this is no longer necessary, since we introduced OxmId)